### DISCUSSION TITLE
「全聯小時達」門市將增至全台 750 家

### DISCUSSION THREAD

**Discussion by AngusGuei** [1101-GS3073-C/Python-Community-GS3073-C#436] [General]
「全聯福利中心在 2021 年初推出自營的「全聯小時達」外送服務平台，並先後與 Uber Eats、foodpanda 兩大業者合作，至今生鮮外送從首波雙北 50 間門市擴展至全台超過 500 間，並提供 3,000 項商品選購、線上線下同價，推動全聯小時達成為「生鮮電商第一品牌」。」

以疫情前全聯在台灣的市場佔有率，轉電商應該成效會很不錯

https://technews.tw/2022/05/27/pxmart-pxgo/

**Comment by YuHengChen54** [comment]
這樣買全聯的東西真的很方便，而且全聯很多東西蠻便宜的哈哈哈

**Comment by BingRong1217** [comment]
這樣就可以很方便的外送生食了耶

**Comment by limitx0** [comment]
全聯的版圖越來越大了，發展得越來越好
外送市場真的是塊很大的餅

**Comment by awehao** [comment]
商業戰爭

**Comment by Chenszuhong** [comment]
全聯東西真的還蠻多的 而且價格也可以接受 但是我比較長使用的是外送家樂福
